app/presenters/collection_show_presenter.rb

```diff
@@ -1,6 +1,6 @@
 class CollectionShowPresenter < CurationConcerns::CollectionPresenter
   include CurationConcerns::Serializers
-  delegate :id, :title, :exhibit_id, :date_created, to: :solr_document, allow_nil: true
+  delegate :id, :title, :exhibit_id, :date_created, :keyword, to: :solr_document, allow_nil: true
```

Comment:
I'm not sure where it's coming from, but I added it because it was throwing test errors. I can try again.